### PR TITLE
Multiple Sessions

### PR DESCRIPTION
**Pull request by mathewscullin**
## Overview

Updated responses for the new changes to the backend API.

## Changes Made

^^ + now catalog number is displayed in the home cell per the Figma.

## Test Coverage

Tested on simulator doing multiple sign-ins with the same account.

### CODE REVIEW

File: CourseGrab/Extensions/UserDefaults+Extensions.swift [hunk 1]
```diff
@@ -19,12 +28,12 @@ extension UserDefaults {
         }
     }
     
-    var areNotificationsEnabled: Bool {
+    var storedDeviceToken: String {
```
Comment (ywsang) [inline CourseGrab/Extensions/UserDefaults+Extensions.swift:31]: was very confused how you were just getting `UserDefaults.standard.storedDeviceToken` in AppDelegate.. and then saw this -- this is cool

**Review by ywsang — APPROVED** [review]
lgtm 🚀

File: CourseGrab/ViewControllers/Home/HomeTableViewCell.swift [hunk 2]
```diff
@@ -124,7 +124,7 @@ class HomeTableViewCell: UITableViewCell {
     
     func configure(for section: Section) {
         self.section = section
-        courseLabel.text = String(section.courseNum)
+        courseLabel.text = String(section.catalogNum)
```
Comment (haiyingweng) [inline CourseGrab/ViewControllers/Home/HomeTableViewCell.swift:127]: yay

**Review by haiyingweng — APPROVED** [review]
🚢

tested before and worked well

**Pull request merged**